product-service/handlers/getProductsById.ts

```diff
@@ -0,0 +1,28 @@
+import { APIGatewayProxyHandler } from 'aws-lambda';
+import 'source-map-support/register';
+import { ProductsService } from '../services/products.service';
+
+export const getProductsById: APIGatewayProxyHandler = async (event, _context) => {
+  const productsService = new ProductsService();
+  const { productId } = event.pathParameters;
+  try {
+    const product = await productsService.getProductById(productId);
+    return {
+      statusCode: 200,
+      body: JSON.stringify(product),
+      headers: {
+        'Access-Control-Allow-Origin': '*',
```

Comment:
These headers can be moved to constants, they repeating in some code parts.

Reply:
Thank you! It's a good point.